packages/moonstone/Heading/Heading.module.less

```diff
@@ -59,6 +59,7 @@
 
 		&.showLine {
 			border-bottom-color: @moon-heading-border-color;
+			padding-bottom: 9px;
```

Comment:
Should this be set outside of `applySkins` with its own variable?

Reply:
I suppose it could be, sure. I'll make that change real quick.

Reply:
Done.